src/main/java/org/twilmes/sql/gremlin/processor/JoinQueryExecutor.java

```diff
@@ -185,8 +186,8 @@ public JoinQueryExecutor(final RelNode node,
                         }
                     }
                 }
-                final Property<Object> property = result.get(tableId).
-                        property(tableIdToTableDefMap.get(tableId).getColumn(simpleFieldName.toLowerCase()).getPropertyName());
+                TableColumn tableColumn = tableIdToTableDefMap.get(tableId).getColumn(simpleFieldName.toLowerCase());
```

Comment:
Sometimes I got null tableColumns here, so original statement will throw NPE in this case.